event/event.go

```diff
@@ -284,15 +291,12 @@ func (ev *event) AssignLab(t *store.Team) error {
 
 func (ev *event) Handler() http.Handler {
 	reghook := func(t *store.Team) error {
-		if ev.labhub.Available() == 0 {
-			return lab.CouldNotFindLabErr
-		}
-
-		if err := ev.AssignLab(t); err != nil {
-			return err
+		if len(ev.labhub.GetLabs())+len(ev.teamQueue) >= ev.GetConfig().Capacity {
```

Comment:
Kind of ugly. Is there a better way?

Reply:
There could be however that has no priority for now. So, lets leave it and merge the branch for now.